tests-system/lobster-trlc/rbt-output-file/default-scenario/expected-output/trlc.lobster

```diff
@@ -0,0 +1,25 @@
+{
+  "data": [
+    {
+      "tag": "req test_default.goodname",
+      "location": {
+        "kind": "file",
+        "file": "./default_file.trlc",
```

Comment:
There is an issue here: on Windows the paths use `\\` instead of `/`. So this test will only succeed on Linux.